packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -360,12 +323,18 @@ export default function App() {
             theme={rainbowkitTheme}
             {...rainbowKitProviderProps}
           >
-            <WelcomeDialog {...welcomeDialogProps} onClose={onClose} />
-            <NetworkReady>
-              <AppContextProvider>
-                <Injector>{isTosAccepted && <AppContent />}</Injector>
-              </AppContextProvider>
-            </NetworkReady>
+            <Header />
```

Comment:
Moved `Header` to here for the correct position but moved it out of `Layout` so it can call wagmi hooks